{"frames":[{"func":"compute_importance(dta2, IV2, DV)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"unique(data[[attribute]])","file":"","line_number":23,"end_line_number":23,"character_number":3,"end_character_number":47},{"func":"data[[attribute]]","file":"","line_number":23,"end_line_number":23,"character_number":3,"end_character_number":47},{"func":"`[[.data.frame`(data, attribute)","file":"","line_number":23,"end_line_number":23,"character_number":3,"end_character_number":47},{"func":"(function(x, i, exact) if (is.matrix(i)) as.matrix(x)[[i]] else .subset2(x, \n    i, exact = exact))(x, ..., exact = exact)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0}],"message":"Error in base::try(gini, silent = TRUE) : object 'gini' not found\n"}